### DISCUSSION TITLE
Your own wallpaper on AoD

### DISCUSSION THREAD

**Discussion by Clust49** [MonwF/customiuizer#440] [Ideas]
Noticed a bug, but I liked it so much when the album picture is not only transferred to the lockscreen, but also to the AOD
So I thought, it would be great to add custom pictures on AOD and even with visualizer. 
(translator)

https://github.com/MonwF/customiuizer/assets/69423075/a3dd9d57-86ec-4aef-9d84-3fcf88da7e17

**Comment by MonwF** [comment]
get the latest test apk. album art should be hidden on aod
